[mod] simple theme: move initilization of document.searxng to searxng.min.js

dalf opened the pull request:
## What does this PR do?

`searxng.head.min.js` focuses on browser features detection nothing more (to add / remove some CSS classes).
The parsing and evaluation of `searxng.head.min.js` blocks the browser rendering (since it is at the top of the page, the page is blank).
Ideally, we should find a way to remove this file.

## Why is this change important?

* We can add as many parameters in `searxng.min.js`.
* If some CSS trick are found, we can remove `searxng.head.min.js`.

For example: 
* `<noscript><link rel="stylesheet" href="searxng.nojs.min.css"></noscript>`
* About touch screen: https://ferie.medium.com/detect-a-touch-device-with-only-css-9f8e30fa1134

## How to test this PR locally?

* `make run`
* check the UI in a desktop browser.
* check the UI in a mobile browser with a  touch screen.

## Author's checklist

<!-- additional notes for reviewiers -->

## Related issues

Related to https://github.com/searxng/searxng/pull/562

dalf committed:
[mod] simple theme: move initilization of document.searxng to searxng.min.js

keep searxng.min.js as small as possible

return42 commented on searx/templates/simple/base.html:
```diff
@@ -19,14 +19,7 @@
   {% endif %}
   {% block styles %}{% endblock %}
   <!--[if gte IE 9]>-->
```
I assume we can drop the IE9 quirks `<!--[if gte IE 9]>-->`  .. / Microsoft has discouraged the use of Internet Explorer as a default browser.

return42 committed:
[mod] simple theme - improve Cumulative Layout Shift (CLS)

1. request `searxng.head.min.js` to add & remove some CSS classes from html tags,
   as soon as possible / best before any other CSS rules are loaded and applied.
2. request `searxng.min.css`
3. request `js/searxng.min.js`
4. request other styles
5. request favicon and other stuff

Signed-off-by: Markus Heiser <markus.heiser@darmarit.de>

return42 committed:
[build] /static

return42 reviewed (approved):
From my side this PR is ready to merge
